credoai/evaluators/identity_verification.py

```diff
@@ -0,0 +1,314 @@
+import pandas as pd
```

Comment:
Add overall docstring to top of the file, so it plays nice with the documentation :)

Reply:
Good idea! Added.